src/g_editor.c

```diff
@@ -4962,8 +4986,7 @@ void g_editor_setup(void)
     class_addmethod(canvas_class, (t_method)canvas_motion, gensym("motion"),
         A_FLOAT, A_FLOAT, A_FLOAT, A_NULL);
     class_addmethod(canvas_class, (t_method)canvas_got_clipboard_contents,
-        gensym("got-clipboard-contents"), A_SYMBOL, A_NULL);          
-
+        gensym("got-clipboard-contents"), A_FLOAT, A_SYMBOL, A_NULL);
```

Comment:
if you make this a `A_DEFSYMBOL`, you could get rid of the dummyargument when sending the `START`/`END` commands.

Reply:
Ah nice, didn't know that. DEF as in default? like an empty string passed if not defined?

Reply:
Done,